components/org.wso2.carbon.metrics.view.ui/src/main/java/org/wso2/carbon/metrics/view/ui/MetricHierarchyDataWrapper.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I fixed license header in master branch. Also available in v.1.2.0: https://github.com/wso2/carbon-metrics/blob/v1.2.0/components/org.wso2.carbon.metrics.view.ui/src/main/java/org/wso2/carbon/metrics/view/ui/MetricHierarchyDataWrapper.java